examples/next-graphql/components/ui/Header.tsx

```diff
@@ -1,6 +1,6 @@
 export default function Header() {
 
-  // @todo: Fetch menusItems from Drupal
+  // @todo: Fetch menuItems from Drupal
```

Comment:
Do we have this effort captured somewhere?

Reply:
Nope we need a tkt , could you @enZane ?

Reply:
Omw